### DISCUSSION TITLE
How to search events with data aquired from other events

### DISCUSSION THREAD

**Discussion by MalteRosskamp** [datalust/seq-tickets#1802] [Help] [answered]
Hey everyone,

i am working on an application which generates a bunch of logs which are linked by the same uuid. Each execution linked by the same uuid generated a summary log at the end which includes an error count. My target is write a query which selects multiple events with the same uuid based on the value of this summary log.

I've attached two pictures which show the two seperate queries, however I have not figured out, how to join them. pls help :)

![2023-03-23_15h21_27](https://user-images.githubusercontent.com/98888704/227234117-94d64ef0-2cf3-49f2-ac8c-977592584e5a.png)
![2023-03-23_15h21_55](https://user-images.githubusercontent.com/98888704/227234154-5fcdeb90-9cbf-46d1-9c76-fecbc15fd2a4.png)

**Comment by nblumhardt** [comment, marked as answer]
Hi! I don't think there's a neat way to do this in one query with Seq today. Joins are one feature we've accounted for in the (very) new query engine, so there's a path to supporting this future, but right now you'll need to use two queries, unfortunately.
